fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/DataServlet.java

```diff
@@ -408,10 +426,13 @@ public void getData(
                     httpResponseMaker
             );
 
-            RequestLog.switchTiming("logRequestMetrics");
-            logRequestMetrics(apiRequest, readCache, druidQuery);
-            RequestLog.record(new DruidFilterInfo(apiRequest.getFilter()));
-            RequestLog.switchTiming(REQUEST_WORKFLOW_TIMER);
```

Comment:
Well, we know a technique for fixing the static dependency, but it's a bigger effort than this PR.
Do we care about timers that have started but not stopped?